freqtrade/freqtradebot.py

```diff
@@ -373,6 +374,33 @@ def execute_buy(self, pair: str, stake_amount: float, price: Optional[float] = N
         order_id = order['id']
         order_status = order.get('status', None)
 
+        if order_status is None:
+            order['status'] = 'open'
+            order['cost'] = stake_amount
```

Comment:
why cost == stake_amount?
amount * price ?

Reply:
stake_amount is used as parameter to api so the same value is saved in the order. Also, the amount is calculated above as amount = stake_amount / buy_limit_requested so it's simply reverse.

Reply:
\+